src/protocol/libp2p/identify.rs

```diff
@@ -197,7 +200,10 @@ impl Identify {
             peers: HashMap::new(),
             public: config.public.expect("public key to be supplied"),
             protocol_version: config.protocol_version,
-            user_agent: config.user_agent.unwrap_or(DEFAULT_AGENT.to_string()),
+            user_agent: config
+                .user_agent
+                .map(|agent| format!("{} {}", agent, AGENT_SUFIX))
+                .unwrap_or(DEFAULT_AGENT.to_string()),
```

Comment:
IMO this should go to substrate, not litep2p, to let developers using litep2p set the user agent they want.